build.gradle

```diff
@@ -1,6 +1,9 @@
-apply plugin: 'java'
-apply plugin: 'application'
-apply plugin: 'eclipse'
+plugins {
+    id 'java'
+    id 'application'
+    id 'eclipse'
+    id 'org.mikeneck.graalvm-native-image' version '1.1.0'
```

Comment:
I'd prefer to not add an additional plugin to the `build.gradle`.
Is it possible to just use a shell script that calls `gradle build` and `native-image`?
Or what's the advantage of the gradle plugin?

Reply:
Yes, I think this is possible. See

https://www.graalvm.org/reference-manual/native-image/BuildConfiguration/#building-native-image-with-java-reflection-example

I will try to create a shell script doing exactly that.

I don't know gradle well enough. Maybe we could use a second `build.gradle` renamed to for example `graalvm-build.gradle`?